docs/getting-started/installation.md

```diff
@@ -49,9 +52,9 @@ In this section, you can read the [System Requirements](/getting-started/require
         bbdd.sid=etendo
         bbdd.port=5432
         bbdd.systemUser=postgres
-        bbdd.systemPassword=syspass
```

Comment:
These are the default values, which are created even with install, I would clarify that if you want to use another user explain it, but otherwise use the default one